### DISCUSSION TITLE
Is there a way to turn off Analytics feature for Hive? Questions about scalability and persistence

### DISCUSSION THREAD

**Discussion by hienqn** [graphql-hive/console#1259] [Help] [answered]
Questions above! If a user is interested only in the Schema Registry features: push schema, pull latest schema, delete subgraph schema, and a user doesn't care about all the metrics going through the graph. Is this possible to turn it off? 

How is composed schema persisted? I saw on the diagram it has only a Redis cache. This is an in-memory database.

**Comment by dotansimha** [comment, marked as answer]
Hi there!
You can use the schema registry feature for free - it comes with the free plan, and also with all the other plans. We do not have any limitations on the scheme registry. 
Please note that some registry features such as conditional breaking changes (based on usage) will not work, if you are not reporting any operations data to Hive. 

Regarding the storage question: schemas pushed to Hive are persisted to the Postgres database, and once composed and valid, it's being pushed to the storage/CDN (on the SaaS solution, we are using a standalone service provided by CloudFlare, to ensure availability at all times. On the self-hosted version it's done by using S3-compatible filesystem storage).
The Redis service is also important because we are using that internally for managing the lifecycle of schema push events - so it is needed for running, but not used as actual storage for schemas.
